File: ProgrammingFundamentals/C#/01.IntroAndBasicSyntax/Lab/P03_EmployeeData/P03_EmployeeData.cs
using System;

namespace P03_EmployeeData
{
    class P03_EmployeeData
    {
        static void Main(string[] args)
        {
            string name = Console.ReadLine();
            int age = int.Parse(Console.ReadLine());
            int id = int.Parse(Console.ReadLine());
            double salary = double.Parse(Console.ReadLine());

            Console.WriteLine($"Name: {name}\r\n" +
                                $"Age: {age}\r\n" +
                                $"Employee ID: " + id.ToString("D8") + 
                                "\r\nSalary: " + String.Format("{0:0.00}", salary));
        }
    }
}

File: ProgrammingFundamentals/Java/01.IntroAndBasicSyntax/Lab/src/P03_EmployeeData.java
import java.util.Scanner;
public class P03_EmployeeData {
    public static void main(String[] args) {

        Scanner console = new Scanner(System.in);
        String name = console.nextLine();
        int age = Integer.parseInt(console.nextLine());
        int id = Integer.parseInt(console.nextLine());
        double salary = Double.parseDouble(console.nextLine());

        System.out.format("Name: %s%nAge: %d%nEmployee ID: %08d%nSalary: %.2f",
                name, age, id, salary);
    }
}
